entrypoint.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh -l
+
+# Files from action repo copied to root dir on image creation
+# Files from consumer repo are located in $GITHUB_WORKSPACE
+cd /
+
+# Login to cluster
+oc login https://console.pathfinder.gov.bc.ca:8443 --token="$INPUT_AUTH_TOKEN"
+
+# Clear previous builds including secrets & volume claims
+make oc-all-clean NAMESPACE=$INPUT_NAMESPACE APP_NAME=$INPUT_APP_NAME
+make oc-persisted-clean NAMESPACE=$INPUT_NAMESPACE APP_NAME=$INPUT_APP_NAME
+
+# Create network security policies, takes time to be applied
+make create-nsp NAMESPACE=$INPUT_NAMESPACE APP_NAME=$INPUT_APP_NAME
+sleep 30s
+
+# Run relevant make command, uses MODE input to select
+case "$INPUT_MODE" in
+  "server")
+    make create-server NAMESPACE="$INPUT_NAMESPACE" APP_NAME="$INPUT_APP_NAME" REPO="https://github.com/$GITHUB_REPOSITORY" BRANCH=master IMAGE_TAG=latest SERVER_PORT="$INPUT_SERVER_PORT"
```

Comment:
```BRANCH=master```
master branch is hard-coded. Shouldn't it be we are trying to merge via pull-request?

Reply:
Added branch input, defaults to master.